examples/udp_server.rs

```diff
@@ -0,0 +1,61 @@
+use mio::net::UdpSocket;
+use mio::{Events, Interests, Poll, Token};
+use std::io;
+
+const IN: Token = Token(0);
+
+fn main() {
+    // Set up a new socket on port 9000 to listen on.
+    let socket = UdpSocket::bind("0.0.0.0:9000".parse().unwrap()).unwrap();
+    // Initialize poller.
+    let mut poll = Poll::new().unwrap();
+    // Register our socket with the token IN (defined above) and an interest
+    // in being `READABLE`.
+    poll.registry()
+        .register(&socket, IN, Interests::READABLE)
+        .unwrap();
+
+    // Prepare a buffer for the number of events we can handle at a time.
+    // Someone might wat to echo really fast so lets give it some size.
+    let mut events = Events::with_capacity(1024);
+    // Initialize a buffer for the UDP datagram
+    let mut buf = [0; 65535];
+    // Main loop
+    loop {
+        // Poll if we have events waiting for us on the socket.
+        poll.poll(&mut events, None).unwrap();
+        // If we do iterate throuigh them
+        for event in events.iter() {
+            // Validate the token we registered our socket with,
+            // in this example it will only ever be one but we
+            // make sure it's valid non the less.
+            match event.token() {
+                IN => loop {
+                    // In this loop we receive from the socket as long as we
+                    // can read data
+                    match socket.recv_from(&mut buf) {
+                        Ok((n, from_addr)) => {
+                            // Send the data right back from where it came from.
+                            let _r = socket.send_to(&buf[..n], from_addr);
```

Comment:
Can you handle the error here (by using `?`).

Reply:
👍